OVERVIEW.md

```diff
@@ -148,6 +148,25 @@ jscs path[ path[...]] --preset=jquery
 
 In order to add/remove preset rules you will need to create a `.jscsrc` config file.
 
+### `--extract`
+With this option you can set file mask from which JavaScript should be extracted.
```

Comment:
Sorry I updated my comment - `set file mask` -> `set the file mask`. Actually the use of `from which` should probably be changed. Maybe `This option checks javascript style in the specified file extensions (currently only html)` or something like that.